lib/libc/string/wcscmp.c

```diff
@@ -41,6 +41,7 @@ __RCSID("$NetBSD: wcscmp.c,v 1.3 2001/01/05 12:13:12 itojun Exp $");
 #endif /* LIBC_SCCS and not lint */
 __FBSDID("$FreeBSD$");
 
+#include <stdint.h>
```

Comment:
I don't think this is needed here. I don't see any new intXX_t or PRxxx being introduced.